pkg/interface/launch/src/js/components/group-filter.js

```diff
@@ -0,0 +1,228 @@
+      ? <template className="dib fr">
+        <p className="dib bg-green2 bg-gray2-d white fw6 ph1 br1 v-mid" style={{ marginBottom: 2 }}>
+          {props.invites.length}
+        </p>
+        <span className="dib v-mid ml1">
+          <img
+          className="v-mid"
+          src="/~launch/img/Chevron.png"
+          style={{ height: 16, width: 16, paddingBottom: 1 }}
+          />
+        </span>
+      </template>
+      : <template className="dib fr" style={{paddingTop: 1}}>
+        <span className="dib v-mid ml1">
+          <img className="v-mid"
+          src="/~launch/img/Chevron.png"
+          style={{ height: 16, width: 16, paddingBottom: 1 }}
+          />
+        </span>
+      </template>;
+
+    let selectedGroups = <div/>
+    let searchResults = <div/>
+
+    if (state.results.length > 0) {
+      let groupResults = state.results.map((group => {
+        return(
+          <li
+          key={group[0]}
+          className="tl list white-d f9 pv2 ph3 pointer hover-bg-gray4 hover-bg-gray1-d inter" onClick={() => this.addGroup(group)}>
+            <span className="mix-blend-diff white">{(group[1]) ? group[1] : group[0]}</span>
+          </li>
+        )
+      }))
+      searchResults = (
+        <div className={"tl absolute bg-white bg-gray0-d white-d pv3 z-1 w-100 ba b--gray4 b--white-d overflow-y-scroll"} style={{maxWidth: "15.67rem", maxHeight: "8rem"}}>
+          <p className="f9 tl gray2 ph3 pb2">Groups</p>
+          {groupResults}
+        </div>
+    )
+    }
+
+    if (state.selected.length > 0) {
+      let allSelected = this.state.selected.map((each) => {
+        let name = each[1];
+        return(
+          <span
+            key={each[0]}
+            className={"f9 inter black pa2 bg-gray5 bg-gray1-d " +
+            "ba b--gray4 b--gray2-d white-d dib mr2 mt2 c-default"}
+          >
+          {name}
+          <span
+          className="white-d ml3 mono pointer"
+          onClick={e => this.deleteGroup(each)}>
+            x
+          </span>
+          </span>
+        )
+      })
+      selectedGroups = (
+        <div className={
+          "f9 gray2 bb bl br b--gray3 b--gray2-d bg-gray0-d " +
+          "white-d pa3 db w-100 inter bg-gray5 lh-solid"
+        }>
+        {allSelected}
+        </div>
+      )
+    }
+
+    return (
+      <div className="ml1 dib">
+        <div className={buttonOpened}
+        onClick={() => this.toggleOpen()}
+        ref={(el) => this.toggleButton = el}>
+        <p className="dib f9 pointer pa1 mw5 truncate v-mid">{currentGroup}</p>
+        </div>
+        <div className={dropdownClass}
+          style={{ maxHeight: "24rem", width: 285 }}
+          ref={(el) => { this.dropdown = el }}>
+          <p className="tc bb b--gray3 b--gray1-d gray3 pv4 f9">Group Select and Filter</p>
+          <a href="/~groups" className="ma4 bg-gray5 bg-gray1-d f9 tl pa1 br1 db no-underline" style={{paddingLeft: "6.5px", paddingRight: "6.5px"}}>Manage all Groups
```

Comment:
<img width="340" alt="image" src="https://user-images.githubusercontent.com/3829764/77808900-98887200-708d-11ea-8fa1-69260f007927.png">
This one shows up as you-visited-this-purple for some reason.

Reply:
You're gonna love this. 
<img width="328" alt="Screen Shot 2020-03-30 at 3 53 37 PM" src="https://user-images.githubusercontent.com/20846414/77955802-d5778300-729e-11ea-881a-d31a4c3a3e8f.png">